src/platforms/php/guides/symfony/index.mdx

```diff
@@ -48,6 +48,12 @@ SENTRY_DSN="___PUBLIC_DSN___"
 ###< sentry/sentry-symfony ###
 ```
 
+<Note>
+
+Performance monitoring integrations related to database, caching, and template rendering are enabled by default. To disable them, please see the [Integrations documentation](/platforms/php/guides/symfony/performance/pm-integrations/).
```

Comment:
A few suggestions:

- Can we be more generic on this 🙏? Since I expect new integrations to come up in the future, this way we won't have to remember that this note needs to be updated to cite all of them
- Wouldn't be better to make this a warning rather than a note? In fact, integrations are hooking into critical paths of the framework and of the vendors and therefore, as I already said, there is a performance penalty. Given that people updating from a previous version or installing the SDK will suddendly see their app performance lower, even if maybe it's not noticeable for most of them, this reason should be enough to make prominent this advise of disabling the tracing feature when unneeded

Reply:
Yes, let's make this more generic.

I'll make the Note a Warning, and add the detail you've provided (succinctly in the warning and more robustly on the Integrations page), thanks.